### ISSUE TITLE
Nyaa Torrents not parsing titles with comments 

### ISSUE THREAD

**Issue by kaiketsuT** [elgatito/script.elementum.burst#113]
After fixing the providers.json for nyaa to be the following:
https://nyaa.si/?q=QUERYEXTRA&f=0&c=0_0&s=seeders&o=desc

i was able to search for files, when I noticed some are missing, upon looking at the common things are through the web browser I noticed that most of not all files that do not show are those that have comments

The site shows how many comments have been made, in the name/title information of the search results and you can sort by this option. But for those that have zero comments, the graphic is not displayed.

In the source code for the results it spends a #comments to the link

I haven't been able to work out what to change in the .json file to succeed in displaying all results

**Comment by elgatito** [comment]
I don't understand your issue. 
You should provide information how to reproduce the issue and what exactly is wrong there.

**Comment by kaiketsuT** [comment]
Sorry i was on mobile when writing this so was unable to give images, or be coherent XD 

the url i listed is the latest format for Nyaa searches for me to get them to show anything in Elementum, that was just information to update if this causes others the same issue. the following is the issue:

in the below image is the result for a search for "Endro" ordered by seeders desc. 
![image](https://user-images.githubusercontent.com/49202230/55625894-f8fa1180-57a1-11e9-8cca-0871c5832352.png)
URL: [https://nyaa.si/?q=endro&f=0&c=0_0&s=seeders&o=desc](https://nyaa.si/?q=endro&f=0&c=0_0&s=seeders&o=desc)

Note the chat/speech bubbles (showing there are comments in the page for that torrent) in the name column (1,1 and 8 respectively) these results will not be displayed in the search in Elementum for the same search string, for those results that do not have this with display and work fine.

in the page source for those with comments the ones with comments with have the following code:

````
<tr class="success">
				<td style="padding:0 4px;">
				<a href="/?c=1_2" title="Anime - English-translated">
					<img src="/static/img/icons/nyaa/1_2.png" alt="Anime - English-translated" style="width: 80px; height: 28px;">
				</a>
				</td>
				<td colspan="2">
					<a href="/view/1131161#comments" class="comments" title="1 comment">
						<i class="fa fa-comments-o"></i>1</a>
					<a href="/view/1131161" title="[HorribleSubs] Endro - 12 [1080p].mkv">[HorribleSubs] Endro - 12 [1080p].mkv</a>
				</td>
				<td class="text-center" style="white-space: nowrap;">
<a href="/download/1131161.torrent"><i class="fa fa-fw fa-download"></i></a>					<a href="magnet:?xt=urn:btih:4ab54d1d049e8a0b523634afb377f79695ebb857&amp;dn=%5BHorribleSubs%5D%20Endro%20-%2012%20%5B1080p%5D.mkv&amp;tr=http%3A%2F%2Fnyaa.tracker.wf%3A7777%2Fannounce&amp;tr=udp%3A%2F%2Fopen.stealth.si%3A80%2Fannounce&amp;tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&amp;tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&amp;tr=udp%3A%2F%2Fexodus.desync.com%3A6969%2Fannounce"><i class="fa fa-fw fa-magnet"></i></a>
				</td>
				<td class="text-center">676.0 MiB</td>
				<td class="text-center" data-timestamp="1553967334" title="5 days 18 hours 18 minutes 32 seconds ago">2019-03-30 17:35</td>

				<td class="text-center" style="color: green;">179</td>
				<td class="text-center" style="color: red;">4</td>
				<td class="text-center">3450</td>
			</tr>
````


and those without will be :

````
<tr class="success">
				<td style="padding:0 4px;">
				<a href="/?c=1_2" title="Anime - English-translated">
					<img src="/static/img/icons/nyaa/1_2.png" alt="Anime - English-translated" style="width: 80px; height: 28px;">
				</a>
				</td>
				<td colspan="2">
					<a href="/view/1129208" title="[HorribleSubs] Endro - 11 [720p].mkv">[HorribleSubs] Endro - 11 [720p].mkv</a>
				</td>
				<td class="text-center" style="white-space: nowrap;">
<a href="/download/1129208.torrent"><i class="fa fa-fw fa-download"></i></a>					<a href="magnet:?xt=urn:btih:e154d587b697cfe7af96ea7723c75a9bc3b44bc5&amp;dn=%5BHorribleSubs%5D%20Endro%20-%2011%20%5B720p%5D.mkv&amp;tr=http%3A%2F%2Fnyaa.tracker.wf%3A7777%2Fannounce&amp;tr=udp%3A%2F%2Fopen.stealth.si%3A80%2Fannounce&amp;tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&amp;tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&amp;tr=udp%3A%2F%2Fexodus.desync.com%3A6969%2Fannounce"><i class="fa fa-fw fa-magnet"></i></a>
				</td>
				<td class="text-center">353.8 MiB</td>
				<td class="text-center" data-timestamp="1553362360" title="1 week 5 days 18 hours 21 minutes 26 seconds ago">2019-03-23 17:32</td>

				<td class="text-center" style="color: green;">131</td>
				<td class="text-center" style="color: red;">0</td>
				<td class="text-center">4980</td>
			</tr>
````

only thing i can really see is the lack of comments in the <td colspan="2"> for those that are displayed in Elementum.

i hope this helps clear things up a little. i am on the FireTV stick gen 2 with Kodi 1.8 and burst 0.0.35

**Comment by danny1990** [comment]
I have the same problem with a different website but i usually get enough results that provide working links. Only sometimes it fails to get any.

Would probably need to edit the python code to make it work but at next update it maybe overwritten.

**Comment by elgatito** [comment]
Will be fixed in the next version.

**Comment by elgatito** [comment]
Please, check v0.0.36.

**Comment by kaiketsuT** [comment]
Can confirm, fixed in v0.0.36. thank you :3
